xds/src/main/java/io/grpc/xds/XdsLoadBalancer.java

```diff
@@ -57,7 +63,14 @@
   private final Helper helper;
   private final LoadBalancerRegistry lbRegistry;
   private final FallbackManager fallbackManager;
+  private final BackoffPolicy.Provider backoffPolicyProvider;
+  private final XdsLoadReportClientFactory lrsClientFactory;
 
+  @Nullable
+  private XdsLoadReportClient lrsClient;
+  @Nullable
+  private XdsLbState xdsLbState;
+  private boolean lrsWorking;
```

Comment:
Seems the sole purpose of this field is to make sure we don't call `startLoadReporting()` and `stopLoadReporting()` unnecessarily. Alternatively, we can relax the requirements on `startLoadReporting()` and `stopLoadReporting()` so that a duplicate call becomes no-op. That would make the code simpler.

Reply:
Ok. I weaker the specification for `XdsLoadReportClient` by removing the preconditions.